vue-frontend/src/components/jobs/CareerView.vue

```diff
@@ -0,0 +1,291 @@
+      <div class="description normal-text">
+        <div class="text-weight">
+          When was the last time you did something for the first time?
+        </div>
+        <div class="mt-5">
+          At Canopas, we frequently do new things that help you leave your
+          comfort zone. And when you leave your comfort zone, you will know what
+          you are capable of and you will never be the same again. If you
+          believe your work should have meaning and want to make an impact in
+          the world with your work, come join us.
+        </div>
+      </div>
+    </div>
+
+    <div class="career-container normal-text mt-5">
+      <div class="career-list-container">
+        <div v-for="career in careers" :key="career" class="career-list">
+          <div
+            class="career-header normal-text text-left"
+            @click="openCareer(career.id)"
+          >
+            <div class="career-icon">
+              <font-awesome-icon
+                class="gradient-career-icon"
+                :icon="career.icon_name"
+              />
+            </div>
+            <div class="career-title">{{ career.title }}</div>
+            <font-awesome-icon class="career-plus-icon" icon="plus" />
+          </div>
+          <collapse-transition>
+            <div
+              v-if="isOpenCareerList && career.id == currentCareerIndex"
+              :key="career.summary"
+            >
+              <div class="career-summary">
+                {{ career.summary }}
+              </div>
+
+              <div class="read-apply-btns">
+                <button class="gradient-border-btn">
+                  <font-awesome-icon
+                    class="fa gradient-career-icon"
+                    icon="align-left"
+                    aria-hidden="true"
+                  />
+                  <span>Read More</span>
+                </button>
+                <button class="gradient-btn">
+                  <font-awesome-icon
+                    class="fa"
+                    icon="check-circle"
+                    aria-hidden="true"
+                  />
+                  <span>Apply Now</span>
+                </button>
+              </div>
+            </div>
+          </collapse-transition>
+          <div class="career-divider" v-if="career.id != careers.length"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script type="module">
+import CollapseTransition from "@ivanv/vue-collapse-transition/src/CollapseTransition.vue";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import axios from "axios";
+import config from "@/config.js";
+
+import { faApple, faAndroid } from "@fortawesome/free-brands-svg-icons";
+import {
+  faGlobe,
+  faBullhorn,
+  faPenNib,
+} from "@fortawesome/free-solid-svg-icons";
+
+export default {
+  data() {
+    return {
+      careers: [],
+      currentCareerIndex: 0,
+      isOpenCareerList: true,
+      previousCareerIndex: null,
```

Comment:
Rename 
currentCareerIndex => currentIndex
isOpenCareerList => openList
previousCareerIndex => previousIndex

Why previousCareerIndex is null? can't we take 0?